src/core/drive/form_submission.ts

```diff
@@ -110,6 +110,7 @@ export class FormSubmission {
       if (token) {
         headers["X-CSRF-Token"] = token
       }
+      headers["Accept"] = "text/vnd.turbo-stream.html"
```

Comment:
Actually, now that this is no longer in the StreamObserver, do we want to read this from a constant like `Turbo.Stream.contentType` that we import?

Reply:
@kaspth is there a way to transform a string constant into the StreamObserver's regular expression?

```js
return /^text\/vnd\.turbo-stream\.html\b/.test(contentType)
```

Reply:
Is that now a `contentType.startsWith?(OUR_CONSTANT)` check, or is it more nuanced than that?

Reply:
Oof, I don't know, but maybe we could do:

```ruby
export default contentType() {
  return "text/vnd.turbo-stream.html"
}

export default contentTypeCheck() {
  return /^text\/vnd\.turbo-stream\.html\b/
}
```

from a single file that both could import. When they're together that way the duplication doesn't matter.

Reply:
I think it'd be fine to switch to `String#startsWith()`. Not as precise as `/\b/` but I don't think it matters here.